packages/nuxt/src/core/plugins/dev-only.ts

```diff
@@ -20,7 +20,7 @@ export const DevOnlyPlugin = createUnplugin((options: DevOnlyPluginOptions) => {
       const { type } = parseQuery(search)
 
       // vue files
-      if (pathname.endsWith('.vue') && (type === 'template' || !search)) {
+      if (pathname.endsWith('.vue') && (meta.framework === 'webpack' ? true : type === 'template' || !search)) {
```

Comment:
```suggestion
      if (pathname.endsWith('.vue') && (meta.framework === 'webpack' || type === 'template' || !search)) {
```

Reply:
:+1:

Reply:
Although my preference goes to this if we need to include more (unlikely though)

```suggestion
      if (pathname.endsWith('.vue') && (['webpack'].includes(meta.framework) || type === 'template' || !search)) {
```

Reply:
we only support webpack and vite for now, i think a PR for rspack is on the way, maybe should we wait for it and change the implementation ?